README.md

```diff
@@ -13,19 +13,19 @@ stable
 Usage
 -----
 
-```
+```shell
 vagrant init nixos/nixos-18.09-i686
 # or
 vagrant init nixos/nixos-18.09-x86_64
 ```
 
 Also have a look at the accompanying nixos vagrant plugin:
-https://github.com/nix-community/vagrant-nixos-plugin
+[https://github.com/nix-community/vagrant-nixos-plugin](https://github.com/nix-community/vagrant-nixos-plugin)
```

Comment:
This change is IMHO not required. You can use following syntax:

```suggestion
<https://github.com/nix-community/vagrant-nixos-plugin>
```

Reply:
You are right. Strangely I thought I had seen my linter having a problem with it. But now that I tested it explicitly, the linter is totally fine with this syntax. Thanks for pointing this out.